### PR TITLE
add cloud init

### PR DESCRIPTION
**Pull request by Roadlyfe**

### CODE REVIEW

File: cloud-init.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#cloud-config
+users:
+  - name: adam
```
Comment (stephenlauck) [inline cloud-init.yaml:3]: How can we make this more dynamic? Are there shell variables we could use here?

File: cloud-init.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    ssh-authorized-keys:
+      - ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOauAfXt5gKJI4uRbpH+ewEFEeyNA5FC9mKd9U+dJVKO adamsmith@Adams-MacBook-Air.local
```
Comment (stephenlauck) [inline cloud-init.yaml:5]: How can we get the key into this file or maybe write the whole cloud-init.yaml each time we create a new set of keys?

File: webserver.sh [hunk 3]
```diff
@@ -52,11 +52,11 @@ then
...
-  multipass launch --name relativepath
+  multipass launch --name relativepath --cloud-init cloud-init.yaml
 fi
 
 # lookup ip address of relativepath vm
 RELATIVEPATH_IP=$( multipass info relativepath | grep IPv4 | tr -s ' ' | cut -d ' ' -f 2 )
 
 # ssh to relative path vm
-ssh -i ./relativepath ubuntu@$RELATIVEPATH_IP
+ssh -i ./id_ed25519 adam@$RELATIVEPATH_IP
```
Comment (stephenlauck) [inline webserver.sh:62]: Is there an shell environment variable we could use here instead?

**Review by stephenlauck — CHANGES_REQUESTED** [review]

**Comment by rwc** [comment]
Reviewed this PR w/ Monday Class and we've combined changes into https://github.com/slashrelativepath/relativepath-cycle-3/pull/23

**Pull request closed without merging**